lib/session/session_test.go

```diff
@@ -52,11 +52,14 @@ func (s *SessionSuite) SetUpTest(c *C) {
 	s.clock = clockwork.NewFakeClockAt(time.Date(2016, 9, 8, 7, 6, 5, 0, time.UTC))
 	s.dir = c.MkDir()
 
-	s.bk, err = lite.New(context.TODO(), backend.Params{"path": s.dir})
+	s.bk, err = lite.NewWithConfig(context.TODO(),
+		lite.Config{
+			Path:  s.dir,
+			Clock: s.clock,
+		},
+	)
 	c.Assert(err, IsNil)
 
-	(s.bk.(*lite.Backend)).SetClock(s.clock)
```

Comment:
This was causing a data race when I switched to using s.clock internally in the backend. This has the same problem as an comment from above - one of the implicitly started backend's processes might access the clock in parallel.